.github/workflows/check.yml

```diff
@@ -18,28 +18,33 @@ jobs:
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-        
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - name: Checkout
         uses: actions/checkout@v2
 
       - name: Set-Up
-        run: sudo apt install -y cmake pkg-config libssl-dev git build-essential clang libclang-dev curl
-
-      - name: Install Rustup
         run: |
           curl https://sh.rustup.rs -sSf | sh -s -- -y
           source ~/.cargo/env
+
+      - name: ⚡ Cache
+        uses: actions/cache@v2
```

Comment:
Good find - looks like this cache system is working better.

Reply:
weird that the plugin didn't work -- was surprised by that